devicesimulation/single-vm/setup.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash -ex
+
+APP_PATH="/app"
+WEBUICONFIG="${APP_PATH}/webui-config.js"
+WEBUICONFIG_SAFE="${APP_PATH}/webui-config.js.safe"
+WEBUICONFIG_UNSAFE="${APP_PATH}/webui-config.js.unsafe"
+ENVVARS="${APP_PATH}/env-vars"
+DOCKERCOMPOSE="${APP_PATH}/docker-compose.yml"
+CERTS="${APP_PATH}/certs"
+CERT="${CERTS}/tls.crt"
+PKEY="${CERTS}/tls.key"
+
+# TODO: move files to Remote Monitoring repositories
+REPOSITORY="https://raw.githubusercontent.com/Azure/pcs-cli/xzsheng/deviceSimlation/devicesimulation/single-vm"
```

Comment:
use target branch 'azure-iot-pcs-simulation'.

Reply:
Currently, the suggested branch is not there. I'll suggest to merge this PR first, then change the branch in a different PR. So we will not break the source branch

Reply:
it is here: https://github.com/Azure/pcs-cli/tree/azure-iot-pcs-simulation

Reply:
fix now pls

Reply:
Fixed